sos/plugins/pacemaker.py

```diff
@@ -105,8 +118,9 @@ def setup(self):
         #   PCMK_logfile=filename
         # specified in the pacemaker start-up environment file.
         pattern = '^\s*PCMK_logfile=[\'\"]?(\S+)[\'\"]?\s*(\s#.*)?$'
-        if os.path.isfile(self.defaults):
-            with open(self.defaults) as f:
+        if os.path.isfile(self.envfile):
+            self.add_copy_spec([self.envfile])
```

Comment:
No need to pass a single-entry list here. The `add_copy_spec()` method accepts either a string, or a list of strings - this should be documented in the method docstring: I'll get that fixed up.